code/index.html

```diff
@@ -1,30 +1,148 @@
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&family@600&=Rozha+One&display=swap" rel="stylesheet">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <title>House plant hotel</title>
   </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+  <body>    
+    <!-- HEADER WITH HERO IMAGE AS BACKGROUND, LOGO, NAVBAR, TEXT AREA & BUTTON -->
+    <header>
+        <div class="hero-image">
+            <div class="container">
+                <div class="navbar-container">
+                    <div class="title">
+                        <h1>HOUSE PLANT HOTEL</h1>
+                    </div>
+        
+                    <div class="hamburger-menu">
+                        <div></div>
+                        <div></div>
+                        <div></div>
+                    </div>
+        
+                    <nav>
+                        <a href="#">HOME</a>
+                        <a href="#">SERVICES</a>
+                        <a href="#">ABOUT</a>
+                        <a href="#">CONTACT & FAQ</a>
+                    </nav>
+                </div>
+            </div>
+            <div class="hero-image-details">
+                <div class="hero-image-text"> 
+                    <span>
+                        <h1>Even house plants need a holiday</h1>            
+                        <h3>Book your houseplants in with us whilst you're away</h3>
+                    </span>
+                </div>
+                <div>
+                    <button>DISCOVER</button>
+                </div>
+            </div>
+        </div>
+    </header>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!-- MAIN SECTION WITH FORM & ACCORDIAN -->
+    <main>
+        <!-- FORM -->
+        <section class="form-parent">        
+            <form action="http://httpbin.org/anything" method="POST">
+                <div class="input-section">
+                    <h3>Subscribe to the house plant hotel newsletter</h3>
+                    <label for="fname">
+                        <input type="text" id="fname" name="fname" placeholder="First name" required>
+                    </label>
+                    <label for="lname">
+                        <input type="text" id="lname" name="lname" placeholder="Last name" required>
+                    </label>
+                    <label for="email">
+                        <input type="email" id="email" name="email" placeholder="Email" required>
+                    </label>
+                </div>
+                
+                <!-- RADIO BUTTON SECTION -->
```

Comment:
Good usage of comment separators

Reply:
Yep helps me to understand what I'm doing :)